lib/fetch/sanitize.js

```diff
@@ -77,9 +77,10 @@ export function sanitizePartner(json) {
     id: json.uid,
     name: get('partner.name'),
     level: get('partner.level'),
-    description: get('partner.description').substr(0, 50),
+    description: get('partner.description'),
```

Comment:
After a discussion with Alex it was decided that it is up to the client to limit the description, not badger brain.